src/visip/action/converter.py

```diff
@@ -41,7 +43,7 @@ def call_format(self, representer, action_name, arg_names, arg_values):
         assert len(arg_names) == 2
         return representer.format(representer.token(arg_names[0]), "[", representer.token(arg_names[1]), "]")
 
-    def _evaluate(self, data_list: dtype.List(dtype.Any()), idx: dtype.Int()) -> dtype.Any():
+    def _evaluate(self, data_list: dtype.List(var_type_t), idx: dtype.Int()) -> var_type_t:
```

Comment:
Create signature explicitly in init. Drop annotation here.

Reply:
Done.